### PR TITLE
[CSL-169] Add browse groups

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/modules/browse.js [hunk 1]
```diff
@@ -249,6 +249,53 @@ class Browse {
...
+   * @example
+   * constructorio.browse.getBrowseGroups({
+   *     filters: {
+   *         group_id: 'drill_collection'
+   *     },
+   *     fmt_options: {
+   *         groups_max_depth: 2
+   *     }
+   * });
+   */
```
[the remark this answers is not in the document]
Comment (esezen) [reply src/modules/browse.js:270]: Between jsdoc and function name?

File: src/modules/browse.js [hunk 2]
```diff
@@ -249,6 +249,53 @@ class Browse {
...
+  getBrowseGroups(parameters) {
+    const fetch = (this.options && this.options.fetch) || fetchPonyfill({ Promise }).fetch;
+    const { serviceUrl } = this.options;
+
```
[the remark this answers is not in the document]
Comment (esezen) [reply src/modules/browse.js:274]: I wanted to separate queryParams assignment + deletion of _dt from the rest of the definitions. Changed to:
```
    const { serviceUrl } = this.options;
    const queryParams = createQueryParams(parameters, this.options);

    delete queryParams._dt;

    const queryString = qs.stringify(queryParams, { indices: false });
```

File: src/modules/browse.js [hunk 3]
```diff
@@ -249,6 +249,53 @@ class Browse {
...
+    const queryParams = createQueryParams(parameters, this.options);
+    delete queryParams._dt;
```
[the remark this answers is not in the document]
Comment (esezen) [reply src/modules/browse.js:276]: Yeah I thought it's weird too but this is the response when I don't remove _dt
```
{ message: 'Invalid parameter: "_dt"' }
```

File: src/modules/browse.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sblaurock) [reply src/modules/browse.js:270]: Oh wait nope you're good - ignore me =)

**Review by sblaurock — APPROVED** [review]
Great work!

**Pull request merged**